lib/liquid/combinators/tags/capture.ex

```diff
@@ -13,6 +13,13 @@ defmodule Liquid.Combinators.Tags.Capture do
   import NimbleParsec
   alias Liquid.Combinators.Tag
 
+  @type t :: [
+          capture: [
+            variable_name: String.t(),
+            parts: [String.t() | Liquid.t()]
```

Comment:
`Liquid.t()` is a list with some tag, variable or String, it is already surrounded by a list and already contains `String.t()` you can put only `parts: Liquid.t()`